content/ember/v3/component-is-visible.md

```diff
@@ -5,60 +5,124 @@ until: '4.0.0'
 since: '3.15'
 ---
 
-In ember earlier versions, we had the concept of 'A controller should never call method or change property on its associated view '( Ember.View class)', instead the view should bind the state of its associated controller.'
+Classic components have a number of APIs to handle the wrapper `div` that they create by default.
+One of them is `isVisible`, which controls if the component is hidden to the end user or not.
+`isVisible` is now deprecated in accordance with [RFC #324](https://github.com/emberjs/rfcs/blob/master/text/0324-deprecate-component-isvisible.md).
 
-So, we were using something like this,
+You can update your component in one of two ways, you can wrap your component's template in an `{{if}}`, or you can use the [`hidden`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) HTML attribute.
```

Comment:
There is also css display and visibility styles, not to mention rendering off screen for sighted users to target screen readers only. aria-hidden should also be mentioned for accessibility in the discussion. Not sure if all this is helpful or needlessly complexifies this discussion.

Reply:
Those are good points. I tried to stay as close as possible to the original mechanically, which apparently does `style="display: none"` which is iffy to me xD Since it's a deprecation guide, I think `aria-hidden` should be mentioned if the original and the migrated solution have different behaviors accessibility-wise?

Reply:
@locks I think your updates look good. I'm inclined to say we should merge it and come back to expand if necessary. What do you think?